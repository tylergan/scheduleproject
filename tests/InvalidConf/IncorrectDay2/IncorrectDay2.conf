every tuesday at 1100 run /bin/date 