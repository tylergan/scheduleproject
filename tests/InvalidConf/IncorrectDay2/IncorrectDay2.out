error in configuration: every tuesday at 1100 run /bin/date
